src/pages/RepoPage/FailedTestsTab/FailedTestsPage/MetricsSection/MetricsSection.tsx

```diff
@@ -317,6 +323,19 @@ interface URLParams {
 const getDecodedBranch = (branch?: string) =>
   !!branch ? decodeURIComponent(branch) : undefined
 
+export const historicalTrendToCopy = (interval?: MeasurementInterval) => {
+  switch (interval) {
+    case 'INTERVAL_1_DAY':
+      return '1 day'
+    case 'INTERVAL_7_DAY':
+      return '7 days'
+    case 'INTERVAL_30_DAY':
+      return '30 days'
```

Comment:
nit: we could remove this line

Reply:
trueee, I prefer the explicit nature though personally, in the event someone comes back and changes the default case for whatever reason

Reply:
oh i mean just line 333 but leave 332

Reply:
ohhhh big facts, I like that

Reply:
Updated in [2fbc6ac](https://github.com/codecov/gazebo/pull/3425/commits/2fbc6acbde7a75436ec61f3870069aee1ec80a3d)